pubsub/subscriptions.js

```diff
@@ -0,0 +1,206 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START auth]
+// By default, gcloud will authenticate using the service account file specified
+// by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use the
+// project specified by the GCLOUD_PROJECT environment variable. See
+// https://googlecloudplatform.github.io/gcloud-node/#/docs/guides/authentication
+var gcloud = require('gcloud');
+
+// Instantiate a pubsub client
+var pubsub = gcloud.pubsub();
+// [END auth]
+
+// [START create_subscription]
+/**
+ * Create a new subscription.
+ *
+ * @param {string} topicName Name of the topic for the new subscription.
+ * @param {string} subscriptionName Name for the new subscription.
+ * @param {Function} callback Callback function.
+ */
+function createSubscription (topicName, subscriptionName, callback) {
+  if (!topicName) {
+    return callback(new Error('"topicName" is required!'));
+  } else if (!subscriptionName) {
+    return callback(new Error('"subscriptionName" is required!'));
+  }
+
+  var options = {
+    reuseExisting: true
+  };
+  pubsub.subscribe(topicName, subscriptionName, options, function (err, subscription) {
+    if (err) {
+      return callback(err);
+    }
+
+    console.log('Created subscription %s to %s', subscriptionName, topicName);
```

Comment:
At the risk of being pedantic: you might want to say "created subscription %s to **topic** %s" instead.